src/Apps/Artwork/Utils/createCollectUrl.tsx

```diff
@@ -0,0 +1,60 @@
+import * as qs from "qs"
+import { data as sd } from "sharify"
+
+export const createCollectUrl = ({
+  dimension,
+  category,
+  sizeScore,
+  artistId,
+}) => {
+  let dimensionRange
+
+  if (dimension === null) {
+    dimensionRange = "*-*"
+  } else if (sizeScore < 1600) {
```

Comment:
Can we use the `dimension` parameter instead of checking the value of `sizeScore`?

Reply:
Good idea